imod_coupler/drivers/dfm_metamod/mf6_wrapper.py

```diff
@@ -122,16 +122,55 @@ def get_river_stages(
         stage = bound[:, 0]
         return stage
 
-    def get_river_flux(
+    def set_correction_flux(
         self,
         mf6_flowmodel_key: str,
         mf6_package_key: str,
```

Comment:
can you add a test for this function in the test_mf6_wrapper  file?

Reply:
done